code/drasil-example/dblpend/lib/Drasil/DblPend/DataDefs.hs

```diff
@@ -122,3 +122,17 @@ forceGQD = mkQuantDef force forceGEqn
 
 forceGEqn :: PExpr
 forceGEqn = vScale (sy mass) (sy acceleration)
+
+-- Vectors velocity definitions
+velVecEqn_1 :: PExpr
+velVecEqn_1 = vec2D
+  (sy angularVel_1 $* sy lenRod_1 $* cos (sy pendDisAngle_1))
+  (sy angularVel_1 $* positionXEqn_1)
+
+velVecEqn_2 :: PExpr
+velVecEqn_2 = vec2D
```

Comment:
Although this is correct, what we would really want here is a definition at the level of *vectors* rather than its components. We might not be able to 'say' that right now in Drasil, but we should at least 'say' it in a comment.